schema.sql

```diff
@@ -3,6 +3,7 @@
 CREATE TABLE animals(
   id                INT GENERATED ALWAYS AS IDENTITY,
   name              VARCHAR(100),
+  species           VARCHAR(255),
```

Comment:
- [ ] You have added a new column directly in the table, but for this practice please 🙏🏽 use the psql terminal to add the new column 👇🏽 

`ALTER TABLE animals ADD species varchar(255);`